Cargo.toml

```diff
@@ -13,3 +13,10 @@ snafu = "0.7"
 structopt = "0.3"
 tokio = { version = "1.4", features = ["full"] }
 tower = {version = "0.4"}
+directories = "4.0.1"
+reqwest = {version = "0.11", default-features = false, features = ["json", "rustls-tls", "stream"]}
+serde = "1.0.144"
+indicatif = "0.17.0"
```

Comment:
Ha neat - look forward to testing this.

Reply:
I know that we didn't wanted to have progress bars in this iteration, but they are easy to implement and they are much better than just waiting for it to upgrade ^^'